mapper/mapper.go

```diff
@@ -241,8 +242,15 @@ func (m *Mapper) buildMessageMapper(message *descriptor.Message, input bool) {
 
 	var oneofMappers []*OneofMapper
 	for _, oneof := range message.Oneofs {
-		oneofMappers = append(oneofMappers, m.buildOneofMapper(oneof, input))
+		var existedMapper *OneofMapper
```

Comment:
have you tested these changes with multiple `oneof`, ie we have not one option but 2, so that we create right graph types